contracts/Utils.sol

```diff
@@ -13,6 +13,21 @@ contract Utils {
     IERC20 internal constant ETH_TOKEN_ADDRESS = IERC20(
         0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE
     );
+    IERC20 internal constant USDT_TOKEN_ADDRESS = IERC20(
```

Comment:
Can add some more comments on top of the contract the reasons that we use the internal constants here?